test/cjs-mocha-compiler.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs = require('fs');
+var traceurAPI = require('../src/node/api.js');
+
+var org = require.extensions['.js'];
+require.extensions['.js'] = function(module, path) {
+  if (!/\/node_modules\//.test(path)) {
+    var content = fs.readFileSync(path, 'utf8');
+    if (shouldSkip(path, content)) {
+      return;
+    }
+
+    // TODO(arv): Reuse parseProlog.
```

Comment:
I extracted parseProlog to src/util/parseProlog.js so that once we have done an npm publish with that this file can then require it. My goal is to read the options so that we can test non default options too. I'll clean up the comment.